src/libs/actions/PersonalDetails.js

```diff
@@ -66,18 +65,15 @@ function getDisplayName(login, personalDetail) {
 }
 
 /**
- * Returns object with first and last name errors. If either are valid,
+ * Returns object with character limit errors. If any are valid,
  * those errors get returned as empty strings.
  *
- * @param {String} firstName
- * @param {String} lastName
- * @returns {Object}
+ * @param {Number} limit
+ * @param {Object[]} toBeValidated
+ * @returns {Object[]}
  */
-function getFirstAndLastNameErrors(firstName, lastName) {
-    return {
-        firstNameError: ValidationUtils.isValidLengthForFirstOrLastName(firstName) ? '' : Localize.translateLocal('personalDetails.error.firstNameLength'),
-        lastNameError: ValidationUtils.isValidLengthForFirstOrLastName(lastName) ? '' : Localize.translateLocal('personalDetails.error.lastNameLength'),
-    };
+function getCharacterLimitErrors(limit, toBeValidated) {
+    return _.map(toBeValidated, x => (x.length <= limit ? '' : Localize.translateLocal('personalDetails.error.characterLimit', {limit})));
```

Comment:
This function sounds more like a validation method so I would suggest moving it to Validation utils.

Reply:
It is but I don't see any point moving it out into a utility file as i don't think it will be reused. I moved it out of the utils for this reason. 
Also it used up more characters using the function with its module namespace than the implementation. Seemed like an over optimisation.